src/software/export/main_exportAnimatedCamera.cpp

```diff
@@ -89,86 +108,174 @@ int main(int argc, char** argv)
     ALICEVISION_LOG_ERROR("The input SfMData file '" << sfmDataFilename << "' is empty.");
     return EXIT_FAILURE;
   }
+  system::Timer timer;
+  std::regex regexFilter;
 
-  sfm::SfMData sfmDataFilter;
-  if(!sfm::Load(sfmDataFilter, sfmDataFilterFilename, sfm::ESfMData::VIEWS))
-  {
-    ALICEVISION_LOG_ERROR("The input SfMData file '" << sfmDataFilterFilename << "' cannot be read.");
-    return EXIT_FAILURE;
-  }
+  if(!viewFilter.empty())
+    regexFilter = std::regex(viewFilter);
```

Comment:
You can convert the user filter into a regex:

```
filterToRegex = std::regex_replace( filterToRegex, std::regex( "\\*" ), "(.*)" );
filterToRegex = std::regex_replace( filterToRegex, std::regex( "\\?" ), "(.)" );
filterToRegex = std::regex_replace( filterToRegex, std::regex( "\\@" ), "[0-9]+" ); // one @ correspond to one or more digits
filterToRegex = std::regex_replace( filterToRegex, std::regex( "\\#" ), "[0-9]" ); // each # in pattern correspond to a digit
```

Reply:
done